homework/01LessonOne.js

```diff
@@ -143,7 +146,9 @@ var notNotUndefined;
 //
 // Explain truthiness and falsiness in your own words. Provide an example for each.
 //
-// Your Answer Goes Here:
+// Your Answer Goes Here: tough question, if a value is equal to another value then it
+// is true. i.e 0 === 0 , 0 is equal to 0.  if it is not then it would be 0 === 1
+// which would be false since a value isn't equal to the other.
```

Comment:
The question here is about **truthiness**. Take a look at this [MDN article on the concept of "truthy"](https://developer.mozilla.org/en-US/docs/Glossary/Truthy).